modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryStorageEngine.java

```diff
@@ -116,6 +116,8 @@ public class PersistentPageMemoryStorageEngine extends AbstractPageMemoryStorage
 
     private volatile ExecutorService destructionExecutor;
 
+    private volatile ExecutorService asyncIoExecutor;
```

Comment:
Please mark that can be `null` (with `@Nullable`) with an explanation why.
I propose to rename it to `asyncFileIoExecutor`.